media/css/pebbles/includes/lib.less

```diff
@@ -243,12 +311,8 @@
 }
 
 
-
-/****************************************************************************/
-// Classes used to hide content from users. Choose wisely!
-
-// Hide visually, but not to screen readers
-.visually-hidden() {
+// Hide an element visually, but not from screen readers
+.a11y-hidden() {
```

Comment:
Hm, fair point. I've just never liked "visually-hidden" because it sounds redundant, I wanted it to be clear this is hidden in an accessible way. But I suppose the comments make that clear enough. I'll change the name back.